### PR TITLE
Cloud 2002.0.18 release

### PR DESCRIPTION
**Pull request by hguthrie**
This contains the updates for the Cloud 2002.0.18 release.
https://magento2.atlassian.net/browse/MAGECLOUD-3401

- [x]   final dev review
- [x]   whatsnew updated with links
- [x]   editor review
- [x]   link check

whatsnew
Updated the Cloud Release notes for ece-tools version 2002.0.18.

- Significant update to the [Cloud Docker development](https://devdocs.magento.com/guides/v2.3/cloud/docker/docker-development.html) section.
- New section explaining the [Docker launch modes](https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-config.html#launch-modes).
- New Docker topic for [Connecting to the database](https://devdocs.magento.com/guides/v2.3/cloud/docker/docker-database.html).
- Changed the top navigation for Cloud by expanding to two columns.
- Added new environment variables.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4448) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4448) before we can accept your contribution.<br/>**3** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: meker12<br/>:white_check_mark: hguthrie<br/>:white_check_mark: shiftedreality<br/>:x: arhiopterecs<br/>:x: NadiyaS

File: guides/v2.1/cloud/docker/docker-config.md [hunk 1]
```diff
@@ -11,23 +11,46 @@ functional_areas:
...
-| ------------- | ---------- | ------------- | ----------------
-| PHP           | `--php`    | 7.1           | 7.0, 7.1, 7.2
-| NGINX         | `--nginx`  | latest        | 1.9, latest
-| MariaDB       | `--db`     | 10            | 10.0, 10.1, 10.2
-| Elasticsearch | `--es`     | 2.4           | 1.7, 2.4, 5.2, 6.5
-| RabbitMQ      | `--rmq`    | 3.5           | 3.5, 3.7
-| Redis         | `--redis`  | 3.2           | 3.0, 3.2, 4.0, 5.0
+Mode is an additional configuration option for the Docker configuration generator (the `docker:build` command). You can launch your Docker environment in one of two modes:
 
+-   **production**—Production mode is the default configuration setting for launching the Docker environment. This option builds the Docker environment in production mode and verifies configured service versions.
```
[the remark this answers is not in the document]
Comment (hguthrie) [reply guides/v2.1/cloud/docker/docker-config.md]: I updated the explanation of Mode ^^

File: guides/v2.1/cloud/docker/docker-quick-reference.md [hunk 2]
```diff
@@ -19,19 +19,34 @@ Use Magento command | `docker-compose run deploy magento <command>`
...
-
+List images | `docker-compose images`
+List containers and ports | `docker-compose ps`, or `docker ps`
 
 ## ece-tools
 
-The following example lists the `{{site.data.var.ct}}` commands that relate to [launching a Docker development environment]({{page.baseurl}}/cloud/docker/docker-config.html):
+Action | Command
+:----- | :------
+Builds the docker environment in production mode by default and verifies configured service versions. | `docker:build`
```
[the remark this answers is not in the document]
Comment (meker12) [reply guides/v2.1/cloud/docker/docker-quick-reference.md]: I had the same thought.  Good info at top that explains the Docker mode setting: 

```_Mode_ is an additional configuration option for the Docker configuration generator (the `docker:build` command). This mode does not affect the Magento mode. It determines the {{site.data.var.ece}} filesystem installation and read-only or read-write behavior.```

Might be useful to add note here, or make "mode" link.

[the remark this answers is not in the document]
Comment (hguthrie) [reply guides/v2.1/cloud/docker/docker-quick-reference.md]: Agreed. I will add the ability to XREF and understand the modes.

**Comment by hguthrie** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

File: guides/v2.1/cloud/docker/docker-development-debug.md [hunk 3]
```diff
@@ -18,27 +18,28 @@ If you use Microsoft Windows, take the following steps before continuing:
...
+    - DEBUG=false
+    - ENABLE_SENDMAIL=false
+    - UPLOAD_MAX_FILESIZE=64M
+    - MAGENTO_ROOT=/app
+    - PHP_ENABLE_XDEBUG=false
+    - PHP_IDE_CONFIG=serverName=magento_cloud_docker
+    - XDEBUG_CONFIG=remote_host=host.docker.internal
+  env_file:
+    - ./docker/config.env
 ```
```
Comment (arhiopterecs) [inline guides/v2.1/cloud/docker/docker-development-debug.md]:
```
 generic:
    image: alpine
    environment:
      - PHP_MEMORY_LIMIT=2048M
      - UPLOAD_MAX_FILESIZE=64M
      - MAGENTO_ROOT=/app
      - PHP_IDE_CONFIG=serverName=magento_cloud_docker
      - XDEBUG_CONFIG=remote_host=host.docker.internal
      - 'PHP_EXTENSIONS=bcmath bz2 calendar exif gd gettext intl mysqli pcntl pdo_mysql soap sockets sysvmsg sysvsem sysvshm opcache zip redis xsl xdebug'
    env_file:
      - ./.docker/config.env 
```

File: guides/v2.1/cloud/docker/docker-config.md [hunk 4]
```diff
@@ -36,6 +61,7 @@ You must have the following software installed on your local workstation:
     -  [php@7.2](https://formulae.brew.sh/formula/php@7.2)
 -  [Composer](https://getcomposer.org)
 -  [Docker](https://www.docker.com/get-started)
+-  [docker-sync](https://docker-sync.readthedocs.io/en/latest/getting-started/installation.html)—file synchronization is required for developer mode
```
Comment (arhiopterecs) [inline guides/v2.1/cloud/docker/docker-config.md]: or [mutagen](https://mutagen.io/documentation/installation/#installation)

**Pull request merged**